js/index.js

```diff
@@ -14,29 +14,82 @@ const siteContent = {
-    "about-h4":"About",
+    "about-h4": "About",
     "about-content": "About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
     "middle-img-src": "img/mid-page-accent.jpg",
-    "services-h4":"Services",
+    "services-h4": "Services",
     "services-content": "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
-    "product-h4":"Product",
+    "product-h4": "Product",
     "product-content": "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
-    "vision-h4":"Vision",
+    "vision-h4": "Vision",
     "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
   },
   "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
+    "contact-h4": "Contact",
+    "address": "123 Way 456 Street Somewhere, USA",
+    "phone": "1 (888) 888-8888",
+    "email": "sales@greatidea.io",
   },
   "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+    "copyright": "Copyright Great Idea! 2018"
   },
 };
 
+
+
+
+
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+// The images
+let cta = document.getElementById('cta-img');
+cta.src = siteContent['cta']['img-src']
+
+let middlePic = document.getElementById('middle-img');
+middlePic.src = siteContent['main-content']['middle-img-src']
+
+
+//Content
+let refLinks = document.querySelectorAll('a');
+refLinks[0].textContent = siteContent['nav']['nav-item-1']
+refLinks[1].textContent = siteContent['nav']['nav-item-2']
+refLinks[2].textContent = siteContent['nav']['nav-item-3']
+refLinks[3].textContent = siteContent['nav']['nav-item-4']
+refLinks[4].textContent = siteContent['nav']['nav-item-5']
+refLinks[5].textContent = siteContent['nav']['nav-item-6']
+refLinks[0].style.color = 'green';
+refLinks[1].style.color = 'green';
+refLinks[2].style.color = 'green';
+refLinks[3].style.color = 'green';
+refLinks[4].style.color = 'green';
+refLinks[5].style.color = 'green';
+
+//Headers
+let awesomeDom = document.getElementById('h1');
+awesomeDom.textContext = siteContent['cta']['h1']
+
+
+//Button
+let startButton = document.getElementById('button');
+startButton.textContent = siteContent['cta']['button']
+
```

Comment:
it would be a good idea to try grabing the whole .text-content and then grab the elements by tagName off of the variable